docs/source/tutorial/initialization.rst

```diff
@@ -0,0 +1,69 @@
+Initialization
+==============
+
+To instantiate ``CitrinationClient``, you will need to provide an
+API key and the host URL of the Citrination site you are trying
+to interact with.
+
+You can find your API key by navigation to ``https://citrination.com``,
+or your private Citrination deployment, logging in, and using the menu
+in the top right to visit your ``Account Settings`` page.
+
+.. attention::
+  Your API allows you (and anyone who knows it) to access your account.
+  Use caution when storing it on your filesystem.
+
+Host URL
```

Comment:
Why a heading for Host URL but not API key?

Reply:
👍